5.5/root/usr/share/container-scripts/mysql/common.sh

```diff
@@ -14,55 +20,13 @@ export MYSQL_FT_MAX_WORD_LEN=${MYSQL_FT_MAX_WORD_LEN:-20}
 export MYSQL_AIO=${MYSQL_AIO:-1}
 
 # Be paranoid and stricter than we should be.
-# https://dev.mysql.com/doc/refman/5.5/en/identifiers.html
+# https://dev.mysql.com/doc/refman/5.6/en/identifiers.html
```

Comment:
I agree and just found out the redirect magically works if we simply omit the version piece.. will do an update...